frontend/front/src/components/SurveyComponent/SurveyComponent.js

```diff
@@ -259,22 +265,19 @@ const SurveyComponentWrapper = forwardRef((props, ref) => {
     return null;
   }, [defaultData, surveyStructure]);
 
+  if (isSurveyLoading) {
+    return <Loader />;
+  }
   return (
     <div ref={ref} style={style}>
-      {surveyStructure && formDefault && activeHome ? (
+      {!surveyStructure && formDefault && activeHome ? (
```

Comment:
I think this logic is wrong, `surveyStructure` needs to exist for the `SurveyComponent` to render.

Reply:
I agree, I think the previous logic was correct.

Reply:
Sorry, I did that to test the error message. I will fix it. Thank you!